java/src/main/c++/vowpalWabbit_learner_VWLearners.cc

```diff
@@ -26,6 +26,20 @@ JNIEXPORT void JNICALL Java_vowpalWabbit_learner_VWLearners_closeInstance(JNIEnv
   }
 }
 
+JNIEXPORT void JNICALL Java_vowpalWabbit_learner_VWLearners_saveModel(JNIEnv *env, jclass obj, jlong vwPtr, jstring filename)
+{ try
+  {
+    const char* utf_string = env->GetStringUTFChars(filename, NULL);
+    std::string filenameCpp(utf_string);
+    VW::save_predictor(*(vw*)vwPtr, filenameCpp);
+    env->ReleaseStringUTFChars(filename, utf_string);
+    env->DeleteLocalRef(filename);
```

Comment:
Can we release the memory outside of the try block (preferably in a finally block).